### PR TITLE
Docker for pcap-miner

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
related: https://github.com/demisto/etc/issues/18521

## Description
A docker file for PCAPMinerv2 might solve the RSA issue.

### CODE REVIEW

File: docker/pcap-miner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+FROM demisto/python3:3.8.3.8715
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
+  && pip install --no-cache-dir -r requirements.txt \
+  && apk del .build-dependencies
+  && apk install tshark
```
Comment (orlichter1) [inline docker/pcap-miner/Dockerfile:9]: @yaakovi added this line (opened a demisto/python3:3.8.3.8715 docker and saw that it installs tshark 3.0.2). Is this the correct place to add it?

File: .dependabot/config.yml [hunk 2]
```diff
@@ -1355,4 +1355,17 @@ update_configs:
...
+    - match:
+        update_type: semver:minor
+  - package_manager: docker
+    directory: /docker/pcap-miner
+    update_schedule: daily
+    automerged_updates:
+    - match:
+        dependency_name: demisto/*
+        update_type: all
+
```
Comment (orlichter1) [inline .dependabot/config.yml:1371]: @yaakovi is this replacement ok? I think it was automatically generated by the docker script.

**Comment by glicht** [comment]
Did you look at: https://github.com/demisto/dockerfiles/blob/master/docker/pcap-http-extractor/Dockerfile 

It seems to be the same

**Comment by orlichter1** [comment]
@glicht I was using that one to develop on but I hit an issue that might have to do with the tshark (Wireshark util) version installed on it. After a talk with @yaakovi and @Itay4 we think that the way to fix it is to install tshark on an alpine docker and not a debian one like the one in pcap-http-extractor.

File: docker/pcap-miner/Pipfile.lock [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        },
+        "sources": [
+            {
+                "name": "pypi",
+                "url": "https://pypi.org/simple",
+                "verify_ssl": true
+            }
+        ]
+    },
+    "default": {},
```
Comment (glicht) [inline docker/pcap-miner/Pipfile.lock:18]: Something seems wrong here. It seems that you are missing the dependencies. Run: pipenv lock and commit the file.

File: .dependabot/config.yml [hunk 4]
```diff
@@ -1355,4 +1355,17 @@ update_configs:
     automerged_updates:
     - match:
         dependency_name: demisto/*
-        update_type: all
\ No newline at end of file
+        update_type: all  - package_manager: python
```
Comment (glicht) [inline .dependabot/config.yml:1358]: Missing new line

File: docker/pcap-miner/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+FROM demisto/python3:3.8.3.8715
+
+COPY requirements.txt .
+
+RUN apk add libxml2 libxslt tshark \
+  && apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
```
Comment (glicht) [inline docker/pcap-miner/Dockerfile:7]: You also need to add here the development packages of libxml2 and libxslt.

**Comment by glicht** [comment]
@orlichter1 status?

**Comment by orlichter1** [comment]
@glicht Looking good locally. Trying on Demisto now. I think it'll be ok now. If so I'll merge. Thanks!

**Comment by orlichter1** [comment]
Working!

**Pull request merged**